web_src/js/features/repo-projects.js

```diff
@@ -190,7 +182,7 @@ export function initRepoProject() {
     if (!columnTitle.val()) {
       return;
     }
-    const url = $(this).data('url');
+    const url = e.target.getAttribute('data-url');
```

Comment:
Interestin, I though we had already forbidden `.data()`, but apparently not.

Reply:
It's possible it's forbidden in plain JS dataset but not in jQuery

Reply:
Ah yes, that might be the case. Should be an easy refactor.